cyclops-ctrl/internal/controller/modules.go

```diff
@@ -577,6 +577,27 @@ func (m *Modules) GetManifest(ctx *gin.Context) {
 	ctx.String(http.StatusOK, manifest)
 }
 
+func (m *Modules) GetRestarted(ctx *gin.Context) {
```

Comment:
Can we rename this function to `Restart` and make the method `POST`? Also can we then read group, version, kind, name and namespace from the request body?

Reply:
Sure. Changed.